### ISSUE TITLE
这个损失计算是不是有问题啊

### ISSUE THREAD

**Issue by vencentDebug** [SCHENLIU/Bert-Hierarchical-Softmax-Chinese-Text-Classification#2]
def _get_loss(self, predicts, label):
        path, _ = config.value_to_path_and_nodes_dict[int(label.data[1])]
        criterion = nn.CrossEntropyLoss()
        if torch.cuda.is_available:
            criterion = criterion.cuda()

        def f(predict, p):
            p = torch.LongTensor([p])
            # convert to cuda tensors if cuda flag is true
            # if torch.cuda.is_available:
            #     p = p.cuda()
            # p = Variable(p)
            return criterion(predict.unsqueeze(0), p)

        loss = list(map(f, predicts, path))
        return torch.sum(torch.stack(loss))
这块好像只计算了label.data[1])一级标签的损失,二级标签没有计算
需要加上二级标签的损失作为才是真正的损失

__________________________________________________________________________________-
另外,
                predict_all_lev1 = np.append(predict_all_lev1, predic_lev1)
                predict_all_lev2 = np.append(predict_all_lev2, predic_lev2)
                labels_all_lev1 = np.append(labels_all_lev1, true[:, 0])
                labels_all_lev2 = np.append(labels_all_lev2, true[:, 1])
你的变量名写错了,一级和二级应该反过来

**Comment by vencentDebug** [comment]
抱歉抱歉,我一级和二级标签和刚刚好与您的相反导致的错误,不好意思
